src/Libraries/Utils/Either.idr

```diff
@@ -5,13 +5,3 @@ module Libraries.Utils.Either
 export
 mapError : (a -> c) -> Either a b -> Either c b
```

Comment:
This is `mapFst` so we can probably get rid of that too.

Reply:
Done, removed the whole module (`Libraries.Utils.Either`)